packages/block-editor/src/components/rich-text/format-toolbar/index.js

```diff
@@ -26,24 +35,18 @@ const FormatToolbar = () => {
 						<Slot
 							name={ `RichText.ToolbarControls.${ format }` }
 							key={ format }
+							bubblesVirtually
 						/>
 					)
 				) }
-				<Slot name="RichText.ToolbarControls">
-					{ ( fills ) =>
-						fills.length !== 0 && (
-							<DropdownMenu
-								icon={ chevronDown }
-								label={ __( 'More rich text controls' ) }
-								controls={ orderBy(
-									fills.map( ( [ { props } ] ) => props ),
```

Comment:
Mapping through fills and using them as "controls" is not something I was expecting. I'm not sure how we can solve this here? Why can't we just use a render Slot in  renderContent of a "Dropdown" component. Anyway, @ellatrix, maybe you can help solve this. (bubbles virtually don't support children as function and can't support it)

Reply:
> Why can't we just use a render Slot in renderContent of a "Dropdown" component.

Could you elaborate?
I'm also not sure how to fix this here. Normally we use a controls array with control objects, but in the case of RichText, components are used. Not sure if we can change that now.

Reply:
Pushed a fix, let me know what you think.

Reply:
How are they now sorted alphabetically?

Reply:
I'm also not sure how the collapsed button can now know if any buttons are active. I guess this would be easier with a controls array.

Another thing I'm noticing is that the buttons have too much space in between them compared to master

Reply:
They are not sorted, I believe they are probably sorted at the "import" level.

The thing is, with  Slot/Fill, we're not supposed to know the shape of the children... So I'd say we were doing it wrong previously or Slot/Fill, is not the right API here.

Reply:
I'll have a look at it now

Reply:
Hm, I tried converting it to control object but that doesn't really work because we're stuck with the React component rendered in the format type edit functions. Maybe we can create a slot and fill for each button, which could be sorted by the title of the format type?

Reply:
It still wouldn't work perfectly though, because some format types could render multiple buttons. That's more of an edge case though.